rake/spec/spec_helper.rb

```diff
@@ -50,7 +50,6 @@ class ST2Spec
     package_has_binaries: {
       st2: %w(st2 st2ctl st2-bootstrap-rmq st2-register-content st2-rule-tester st2-run-pack-tests
               st2-trigger-refire st2 st2-self-check st2-track-result
-              st2-validate-pack-config st2-check-license
```

Comment:
Was `st2-validate-pack-config` removed by mistake here?

Reply:
Yes sorry, fixing it now